corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -432,6 +432,7 @@ Formplayer.ViewModels.CloudCareDebugger = function() {
         $.publish('formplayer.' + Formplayer.Const.FORMATTED_QUESTIONS, function(resp) {
             self.formattedQuestionsHtml(resp.formattedQuestions);
             self.instanceXml(resp.instanceXml);
+            self.evalXPath.autocomplete(resp.questionList);
```

Comment:
ehh as long as the other PR is merged, the latest formplayer always gets put on prod. though would be nice to fail if someone changes the api for some reason. can update